clipmenud

```diff
@@ -152,6 +152,10 @@ while true; do
         clipnotify &
         _CM_CLIPNOTIFY_PID="$!"
         wait "$_CM_CLIPNOTIFY_PID"
+        if [[ $? -ne 0 ]]; then
```

Comment:
No need to manually test $?, just do `wait ... || sleep 10` with the comment above. And it would be great to have a space between the octothorpe and the first word :-)

Reply:
Done. Also, I learned a new word.